extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -222,6 +237,37 @@ const stateComparisonFunctions = {
     (old, curr) => old.element !== curr.element || old.state !== curr.state,
   [StateProperty.NAVIGATION_PATH]: (old, curr) => old.length !== curr.length,
   [StateProperty.PAGE_IDS]: (old, curr) => old.length !== curr.length,
+  [StateProperty.INTERACTIVE_RESULTS_STATE]: (old, curr) => {
+    return old.finished != curr.finished;
+  },
+};
+
+/**
+ * Gets the map of quizzes responded and sets the correct values on the template.
+ * @param {!Object} data
+ * @return {!InteractiveResultsDef}
+ */
+const processInteractiveResults = (data) => {
```

Comment:
Let's move this code to the `story-interactive.js`, we're trying to keep any business logic out of the store-service.

Reply:
Sure, my thoughts were that this logic processes the interactives and it's easy to make a stateComparisonFunction. When I move this into amp-story-interactive.js, the comparison function will be more complex but makes sense